tests/calendars/test_cfe_calendar.py

```diff
@@ -40,3 +40,15 @@ def test_2016_early_closes(self):
         )
         self.assertEqual(12, market_close.hour)
         self.assertEqual(15, market_close.minute)
+
+    def test_adhoc_holidays(self):
+        # hurricane sandy: oct 29 2012, oct 30 2012
+        # national days of mourning:
+        # - apr 27 1994
+        # - june 11 2004
+        # - jan 2 2007
+        for day in ["1994-04-27", "2004-06-11", "2007-01-02",
+                    "2012-10-29", "2012-10-30"]:
```

Comment:
Do you think it's worth testing these separately, or should we fold these dates into `test_2016_holidays` above?